### PR TITLE
Adds [Concept Entry] Sklearn Gaussian Processes 

### PR DESCRIPTION
**Pull request by negarvahid**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [ ] All writings are my own.
- [ ] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [ ] I have performed a self-review of my own writing and code.
- [ ] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [ ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/sklearn/concepts/gaussian-processes/gaussian-processes.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+Title: 'Gaussian Processes'
+Description: 'Gaussian Processes are a supervised learning algorithm that predicts outcomes as distributions, assuming that any set of input points follows a joint Gaussian distribution.'
```
Comment (SaviDahegaonkar) [inline content/sklearn/concepts/gaussian-processes/gaussian-processes.md:3]:
``` suggestion
Description: 'Gaussian Processes is a supervised learning framework. that predicts outcomes as distributions, assuming that any set of input points follows a joint Gaussian distribution.'
```
The use of the word `framework` would be more efficient than using the word `algorithm` as the term algorithm suggests that the Gaussian Processes are a single algorithm which is not accurate they are a framework that can be used in various supervised learning models.

File: content/sklearn/concepts/gaussian-processes/gaussian-processes.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+# Fit the model on the training data (X) and labels (y)
+gp_classifier.fit(X, y)
+
+# Predict class probabilities for new data points
+y_prob = gp_classifier.predict_proba(X)
+
+
+```
+
```
Comment (SaviDahegaonkar) [inline content/sklearn/concepts/gaussian-processes/gaussian-processes.md:56]:
```suggestion


```

```
Remove the extra space here.

**Review by SaviDahegaonkar — CHANGES_REQUESTED** [review]
Hey Author,
I had a look at your entry and suggested some changes to you please make them asap also add an example section and a codebyte example section as mentioned in the [issue](https://github.com/Codecademy/docs/issues/4896) related to this PR.

Thanks,
Savi

**Comment by SaviDahegaonkar** [comment]
Hey @negarvahid , Have you implemented the changes I suggested you, if not please let me know.


Thanks,
Savi

**Comment by negarvahid** [comment]
> Hey @negarvahid , Have you implemented the changes I suggested you, if not please let me know.
> 
> Thanks, Savi

Hi there @SaviDahegaonkar, I somehow had never gotten an email notification and wasn't aware you had already gone through my submission. I will submit an updated version this week. Thanks

**Comment by SaviDahegaonkar** [comment]
> > Hey @negarvahid , Have you implemented the changes I suggested you, if not please let me know.
> > Thanks, Savi
> 
> Hi there @SaviDahegaonkar, I somehow had never gotten an email notification and wasn't aware you had already gone through my submission. I will submit an updated version this week. Thanks

Yes, please make the changes asap. So I can get your PR merged.

**Review by SaviDahegaonkar — APPROVED** [review]
Hey @negarvahid ,
There is no need to make the changes; I have already approved this PR. Thank you for contributing to Docs. Also, I want to suggest that you please try to mention the issue that is closed by a particular PR every time you push a new PR. It makes us easy to review a PR.

**Comment by mamtawardhani** [comment]
Hey @negarvahid, this PR has been raised from the main branch. Can you create a feature branch and raise the issue?
Codecademy requires authors to create a feature branch to merge entries. 
Please do it at your earliest convenience, thank you! 😃

**Comment by mamtawardhani** [comment]
Hey @negarvahid I'll be closing this PR for now, please re-raise it from the feature branch in the next one week.
Thank you! 😄

**Pull request closed without merging**